carbonmark/lib/actions.ts

```diff
@@ -294,9 +294,9 @@ export const getProjectInfoFromApi = async (
       key: project.key,
       projectID: project.projectID,
       name: project.name,
-      methodology: project.methodology,
+      methodology: project.methodologies[0].id,
       vintage: project.vintage,
-      category: project.category?.id || "Other",
+      category: project.methodologies[0].category || "Other",
```

Comment:
To reduce boilerplate and to have deep object constructions in one place
you can use the getter `getCategoryFromProject`  everywhere where you use 
`!!project.methodologies[0]?.category && project.methodologies[0].category` 

And create a second getter `getCategoryIdFromProject` to retrieve the category ID too.
`project.methodologies[0].id`